test/cases/variables/conf/otel_drop_in_support.conf

```diff
@@ -0,0 +1,27 @@
+# "/datadog-tests" is a directory created by the docker build
+# of the nginx test image. It contains the module, the
+# nginx config, and "index.html".
```

Comment:
I'm a bit lost on how this is testing otel ? We add aliases for a bunch of otel directives but we are not really testing them.

Reply:
Quite simple. It uses `$opentelemetry_trace_id` and `$opentelemetry_span_id` variable. Those variables are aliases on `$datadog_trace_id` and `$datadog_span_id` respectively. This test ensures the new `$opentelemetry_` variables are defined and returns the expected value.

Reply:
Yes but we are not testing directive alias

Reply:
Oh that, indeed, good catch.

Reply:
I will add tests in another PR. I want to release a new version at best next week.